sigma/backends/loki/loki.py

```diff
@@ -227,38 +194,70 @@ class LogQLBackend(TextQueryBackend):
     filter_chars: ClassVar[str] = ""
 
     field_replace_pattern: ClassVar[Pattern] = re.compile("[^a-zA-Z0-9_:]+")
-    field_null_expression: ClassVar[str] = "{field}=``"
 
-    # LogQL does not support wildcards, so we convert them to regular expressions
-    wildcard_multi: ClassVar[
-        str
-    ] = "*"  # Character used as multi-character wildcard (replaced with .*)
-    wildcard_single: ClassVar[
-        str
-    ] = "?"  # Character used as single-character wildcard (replaced with .)
+    negated_line_filter_operator: ClassVar[Dict[str, str]] = {
+        "|=": "!=",
+        "!=": "|=",
+        "|~": "!~",
+        "!~": "|~",
+    }
+
+    currentTemplates: ClassVar[Union[bool, None]] = None

```

Comment:
We don't have camel case in Python.
```suggestion
    current_templates: ClassVar[Union[bool, None]] = None
```